docs/reST/ref/rect.rst

```diff
@@ -401,6 +401,124 @@
 
       .. ## Rect.collidelistall ##
 
+   .. method:: collideobjects
+
+      | :sl:`test if an objects in a list intersects`
+      | :sg:`collideobjects(rect_list) -> object`
+      | :sg:`collideobjects(obj_list, key=func) -> object`
+
+      Test wheter the rectangle collides with any object in the sequence.
```

Comment:
> Test wheter the rectangle collides with any object in the sequence.

Should be 'Test **whether** the rectangle...'

Reply:
thanks for spotting